testplan/testing/base.py

```diff
@@ -129,9 +128,11 @@ class Test(Runnable):
     filter_levels = [filtering.FilterLevel.TEST]
 
     def __init__(self, **options):
-        validate_display_name(
-            options.get("name"), MAX_MULTITEST_NAME_LENGTH, "MultiTest name"
-        )
+        test_name = options.get("name", "")
+        if isinstance(test_name, str) and ":" in test_name:
```

Comment:
why are we checking test_name is str here? We have already validated that in cfg schema?